docs/source/programming_resources/tutorial_specific/blocks/blocks_reference/Blocks-Reference-Material.rst

```diff
@@ -1,5 +1,5 @@
-Blocks Reference Materials
-==========================
+Blocks Reference Materials :bdg-warning:`Blocks`
+=================================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".